packages/common/src/ui/layout/tables/DataTable.tsx

```diff
@@ -118,29 +116,28 @@ export const DataTable = <T extends DomainObject>({
             );
           })}
         </TableBody>
-        <TableFooter
-          sx={{
-            flex: 1,
-            display: 'flex',
-            flexDirection: 'column',
-            position: 'sticky',
-            insetBlockEnd: 0,
-            backgroundColor: 'white',
-          }}
-        >
-          <TableRow>
-            {pagination && onChangePage && (
-              <PaginationRow
-                page={pagination.page}
-                offset={pagination.offset}
-                first={pagination.first}
-                total={pagination.total ?? 0}
-                onChange={onChangePage}
-              />
-            )}
-          </TableRow>
-        </TableFooter>
       </MuiTable>
+      <Box
+        sx={{
+          flex: 1,
```

Comment:
```suggestion
          flex: 0,
```

the `flex-grow` here is causing this:

![table oddity](https://user-images.githubusercontent.com/9192912/147181062-8e19cf75-47d9-49e3-8f8d-fc96807f3efc.gif)

The footer is taking up half the space and is hiding some rows in the table. I would change to `flex: 0` or explicitly set `flex-grow: 0`. this does mean that when you have a small table, the pagination is at the bottom of the table, which I think you were trying to avoid? I actually like the pagination to be close to the table, so it suits me 😄  the pagination is still sticky, just not at the bottom of the page 🤷‍♂️

Reply:
I've changed to flex: 0.

Now another problem which i'm not sure about.. when on a smaller screen that needs horizontal scroll, the pagination row stretches the full width 😢

Reply:
oh! balls.

that's the sort of thing I had the grid in place for, to get the 'group by' switch to be visible even when there's horizontal scroll. there are just so many combinations of things to test with this thing!!

Reply:
Yeah! Ugh!

Side note - I think the group by item toggle 'dissapearing' when you scroll is actually quite neat and wondered if all the filters could go in a row like that 😁